Website/src/rendering/src/components/Vendors/VendorsGrid.tsx

```diff
@@ -28,7 +41,7 @@ const VendorsGrid = (props: VendorsGridProps): JSX.Element => {
       </Link>
     ));
 
-  return (
+  return hasVendors ? (
```

Comment:
To reduce the amount of logic in the main JSX return statements, could the return statements of all the modified components be reworked like this?:

```typescript
const pageEditingMissingDatasource = !hasVendors && isPageEditing && (<p>Missing Datasource Item</p>);

const vendorsGrid = hasVendors && (
  <section ...>
    ...
  </section>
);

return (
  <>
    {vendorsGrid}
    {pageEditingMissingDatasource}
  </>
);
```